GruntFile.js

```diff
@@ -5,7 +5,7 @@ require('load-grunt-tasks')(grunt);
 var branch=process.env.sourcebranch;
 var folder=grunt.option('folder')||branch||'.';
 var solutionjsonfiles = [`${folder}/**/*.json`,`!${folder}/node_modules/**/*.json`];
-var createUIDefinition=`${folder}/createUIDefinition.json`;
+var createUIDefinition=`${folder}/createUiDefinition.json`;
```

Comment:
is createUIDefinition and createUiDefinition going to be confusing later?

Reply:
I agree, however we won't be able to change it in the https URI as the URI is correct with the name being `CreateUIDefinition.MultiVm.json`.